openapi/index.yaml

```
@@ -36,6 +36,23 @@ paths:
           description: Service unavailable.
           schema:
             $ref: "#/definitions/ProblemJson"
+    get:
+      operationId: GetEligibilityCheck
```

We usually use lowerCamelCase

```suggestion
      operationId: getEligibilityCheck
```